src/apps/companies/controllers/edit.js

```
@@ -62,10 +75,13 @@ async function renderForm (req, res) {
     .breadcrumb(res.locals.company ? 'Edit' : 'Add')
     .render('companies/views/edit', {
       isForeign,
+      onOneList: onOneList(req, res),
+      companyDetails: res.locals.company ? transformCompanyToView(res.locals.company) : {},
       businessTypeLabel,
       showTradingAddress,
       showCompanyNumber: businessType === UK_BRANCH_OF_FOREIGN_COMPANY_ID,
       heading: getHeading(res.locals.company, isForeign),
+      oneListEmail: config.oneList.email,
```

what if that value isn't there?  `get(config, 'onelist.email')` maybe?

---

If the value isn't there we want it to fail badly so that we know instead of suppress it silently.

---

In that case wrap this all in a try catch and in the catch call `next(error)`